include/libcyphal/transport/udp/transport.hpp

```diff
@@ -46,7 +55,7 @@ class TransportImpl final : public IUdpTransport
     TransportImpl(Spec,
                   cetl::pmr::memory_resource&            memory,
                   IMultiplexer&                          multiplexer,
-                  libcyphal::detail::VarArray<IMedia*>&& media_array,
+                  libcyphal::detail::VarArray<IMedia*>&& media_array,  // NOLINT
```

Comment:
this is temporary - there is no implementation (yet), so no `std::move` of the `media_array`).

Reply:
It is best to comment on that directly in code. For example, a good practice is to always provide a very short explanation right after `NOSONAR`. We even have a script to enforce that: https://github.com/Zubax/dyshlo/blob/main/tools/check_nolint.sh